app/providers/notify_provider.rb

```diff
@@ -0,0 +1,32 @@
+class NotifyProvider
+  attr_accessor :client, :template_id
+
+  def self.call(*args)
+    new.call(*args)
+  end
+
+  def initialize(config: EmailAlertAPI.config.notify)
+    api_key = config.fetch(:api_key)
+    base_url = config.fetch(:base_url)
+
+    self.client = Notifications::Client.new(api_key, base_url)
+    self.template_id = config.fetch(:template_id)
+  end
+
+  def call(address:, subject:, body:, reference:)
+    client.send_email(
```

Comment:
Do we need to pass back the return value of this?

Reply:
We don't currently use the return value of this method call.

Do you think we should return something?

Reply:
Oh, I guess now that we're using `reference` instead of relying on Notify IDs it doesn't matter.